modules/caddytls/connpolicy.go

```diff
@@ -941,6 +941,31 @@ func (l *LeafCertClientAuth) Provision(ctx caddy.Context) error {
 	return nil
 }
 
+// UnmarshalCaddyfile implements caddyfile.Unmarshaler.
+func (l *LeafCertClientAuth) UnmarshalCaddyfile(d *caddyfile.Dispenser) error {
+	d.NextArg()
+	for nesting := d.Nesting(); d.NextBlock(nesting); {
+		modName := d.Val()
+		if !strings.HasPrefix(modName, "load_") {
+			return d.Err("expected a leaf certificate loader module name prefixed with `load_`")
+		}
+		modName = strings.TrimPrefix(modName, "load_")
```

Comment:
@francislavoie, what do you think of this from Caddyfile UX perspective?

Reply:
Do we need `load_` at all?

Reply:
I thought I should future-proof it in case other parameters are added besides the loaders. I thought the namespace will be clobbered if I don't distinguish them with a prefix or nest them into a block. I don't know...

Reply:
The module name is already prefixed by `tls.leaf_cert_loader.`. Are there other "kinds" (?) of modules that go in this namespace that aren't loading?

Reply:
Shouldn't it be like `verifier leaf file <filename>` etc?

Reply:
I'm saying what if `leaf` has more parameters in the future, not the loader modules. Anyways, looking at what `VerifyClientCertificate` does, I can't imagine any new behavioral parameters that may be added in the future.

The format `verifier leaf file <filename>` doesn`t work because it blocks the use multiple loaders, which is currently possible. In other words, this is now possible:

```
verifier leaf {
    file file-1.pem file-2.pem
    file more-cert.pem
    folder /path/to/cert/repo
}
```
The one-liner disallows this use.

Reply:
I've removed the `load_` prefix. I've also made it accept either in-line or block, but not both. All good?